packages/opal-common/opal_common/schemas/sources.py

```diff
@@ -4,7 +4,7 @@
 
 
 class SourceAuthData(BaseModel):
-    auth_type: str = Field(..., description='Authentication type (user/pass, ssh keys, etc)')
+    auth_type: str = Field("none", description='Authentication type (user/pass, ssh keys, etc)')
```

Comment:
what the "none" type is for? public repos?
if so, we need to test that it actually works with public repos.